### PR TITLE
#2061 Add ability to award certificates from the admin

### PR DESCRIPTION
**Pull request by stenington**
This PR reworks the admin participants page certificates section, and adds the ability to generate certificates. A few screenshots to help explain:

### Screenshot 1:
Judge/mentor with no current certificates
Screen Shot 2019-04-03 at 11 49 28 AM

The participants page shows which certificates will be generated by the rake task.

### Screenshot 2:
I used the controls to generate a Head Judge certificate (instead of the Judge Advisor the system thought was right)
Screen Shot 2019-04-03 at 11 49 40 AM

Using the controls to award a certificate kicks off a `CertificateJob` that right now admins don't have a way to monitor. But if everything goes accordingly, the certificate will eventually show up.

### Screenshot 3:
The participants page shows that Head Judge has been generated, and a Mentor Appreciation certificate will be generated. 
Screen Shot 2019-04-03 at 11 49 46 AM

`DetermineCertificates` is smart enough to know that judge certificates are mutually exclusive, so it no longer wants to award Judge Advisor. Clicking *See certificate* takes you to the same page as clicking on a certificate from the datagrid.

### CODE REVIEW

**Comment by stenington** [comment]
Another screenshot, this time for a student who didn't qualify for any certificates to show that use case:

Screen Shot 2019-04-03 at 12 04 24 PM

**Comment by stenington** [comment]
I'm not sure this merged perfectly with #2001, I'm making one more change.

**Comment by stenington** [comment]
@adrichey Ok, this should be ready for review.

File: app/models/certificate.rb [hunk 1]
```diff
@@ -9,7 +9,15 @@ class Certificate < ApplicationRecord
   mount_uploader :file, FileProcessor
 
   scope :judge_types, -> {
-    where(cert_type: CERTIFICATE_TYPES.select { |t| t.include?('judge') })
```
Comment (adrichey) [inline app/models/certificate.rb]: I really like this change.

**Pull request merged**